apps/user/tests/api.spec.e2e.ts

```diff
@@ -90,4 +90,17 @@ describe("UserController (e2e)", () => {
     expect(typeof response.body.access_token).toBe("string");
     expect(response.body.access_token.length).toBeGreaterThan(100);
   });
+
+  it("should return 204 when a user is deleted successfully", () => {
+    function userToDelete() {
+      return {
+        email: ``, // Replace with existing user email from keycloak
+      };
+    }
+    const newUser = userToDelete();
+    return supertest(app.getHttpServer())
```

Comment:
Here is the problem, you don’t need to do supertest after, just call that function directly